src/templates/react-native/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
don't we need this?
```
"settings": {
    "import/internal-regex": "^common|^lib/|^modules/|^pages/"
  }
```

Also, even tho I usually have some rules that I disable almost every time, I feel that we shouldn't disable them by default here, unless there's a very valid reason.